nightfall-optimist/src/routes/block.mjs

```diff
@@ -26,68 +44,66 @@ router.post('/check', async (req, res, next) => {
   }
 });
 
-router.get('/make-now', async (req, res, next) => {
+/**
+ * @openapi
+ *  /block/make-now:
+ *    get:
+ *      security:
+ *        - ApiKeyAuth: []
+ *      tags:
+ *      - Block
+ *      summary: Make block.
+ *      description: Responsible to call the function that will generate a new block.
+ *      parameters:
+ *        - in: header
+ *          name: api_key
+ *          schema:
+ *            type: string
+ *            format: uuid
+ *          required: true
+ *      responses:
+ *        200:
+ *          $ref: '#/components/responses/SuccessBlockCreated'
+ *        401:
+ *          $ref: '#/components/responses/Unauthorized'
+ *        500:
+ *          $ref: '#/components/responses/InternalServerError'
+ */
+router.get('/make-now', auth, async (req, res, next) => {
   try {
-    logger.debug(`block make-now endpoint received GET`);
     setMakeNow();
     res.send('Making short block');
   } catch (err) {
     next(err);
   }
 });
 
-router.get('/transaction-hash/:transactionHash', async (req, res, next) => {
```

Comment:
Why is this endpoint removed?

Reply:
It is unused

Reply:
Aren’t these functions useful. Maybe not being used now, but they could. I have not knowledge to confirm, but I wouldn’t remove it without talking to the one that added them in the first place.

Reply:
Perhaps we can ask on Slack about the 2 endpoints

Reply:
Resolving the conversation since people seem to be okay with these removals